### ISSUE TITLE
Participles

### ISSUE THREAD

**Issue by AleksandrsBerdicevskis** [SwedishJewellery/discussions#4]
It seems that in UD-Talbanken the original participles (PC) were tagged as verbs if they had a dependent _bli_ (via AUX) and as adjectives otherwise (even in constructions with _vara_). Is it a temporary conversion solution or is it meant to be a permanent one?
In UD-Lines, the verb tag seems to be used more broadly. 
In Koala, as far as I understand, the participles are always adjectives.

**Comment by gerlofbouma** [comment]
> In Koala, as far as I understand, the participles are always adjectives.

Yes, or nouns. But never verbs or participles (no such pos in Koala)

> It seems that in UD-Talbanken the original participles (PC) were tagged as verbs if they had a dependent bli (via AUX) and as adjectives otherwise (even in constructions with vara). Is it a temporary conversion solution or is it meant to be a permanent one?

This is in the Swedish UD guidelines (https://universaldependencies.org/sv/index.html#tags) so I guess it should ultimately apply to UD-Lines as well. It is my understanding that this is related to the wish to have a way to identify passives in UD, but a UD contributor should comment on the motivation for this.
